ajax/libs/cleanslate/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cleanslate",
+  "filename": "cleanslate.min.css",
+  "version": "0.10.1",
+  "homepage": "http://cleanslatecss.com",
+  "description": "Extreme CSS reset stylesheet",
+  "keywords": [
+    "reset css"
```

Comment:
@pvnr0082t Sorry I didn't read through this file. I amend the package.json, please review this PR again. Thanks.